File: 2025-1/sesion-de-clase-18/SoftProg/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/model/clientes/Cliente.java
package pe.edu.pucp.inf30.softprog.model.clientes;

import pe.edu.pucp.inf30.softprog.model.Persona;

/**
 *
 * @author eric
 */
public class Cliente extends Persona {
    private Categoria categoria;
    private double lineaCredito;
    
    public Categoria getCategoria() {
        return categoria;
    }

    public void setCategoria(Categoria categoria) {
        this.categoria = categoria;
    }

    public double getLineaCredito() {
        return lineaCredito;
    }

    public void setLineaCredito(double lineaCredito) {
        this.lineaCredito = lineaCredito;
    }
}

File: 2025-2/sesion-de-clase-21/dotnet/SoftProgModelo/Clientes/Cliente.cs
using PUCP.SoftProg.Modelo.RRHH;

namespace PUCP.SoftProg.Modelo.Clientes {
    public class Cliente : Persona {
        public double LineaCredito { get; set; }
        public CategoriaCliente Categoria { get; set; }
        public CuentaUsuario CuentaUsuario { get; set; }
    }
}